scopes/scope/scope/ui/menu/menu.tsx

```diff
@@ -1,10 +1,25 @@
 import { Menu, MenuProps } from '@teambit/ui-foundation.ui.menu';
-import React from 'react';
+import { ScopeContext } from '@teambit/scope'; // we need to separate the provider and context from aspect. is this ok for now?
```

Comment:
extracting scopeContext to its own component will only work inside of `bit-bin` (but not in `symphony`). So I think it's reasonable to use ScopeContext directly from `@teambit/scope`, and it will force us to use it only in bit extensions, and not in smaller ui components (which is probably good)